build.gradle

```diff
@@ -23,9 +23,9 @@ configurations {
 
 dependencies {
     compile localGroovy()
-    def cargoVersion = '1.6.8'
+    def cargoVersion = '1.7.7'
```

Comment:
I'd suggest you upgrade to Codehaus Cargo 1.7.9, which has a second fix for https://codehaus-cargo.atlassian.net/browse/CARGO-1482 and is beneficial for the Gradle Cargo plugin